### PR TITLE
19023 Implemented Involuntary Dissolution Tile

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #: https://github.com/bcgov/entity/issues/19023

*Description of changes:*
- Implemented involuntary Dissolution to staff dashboard
- Implemented feature flag
- Created skeleton for the involuntary dissolution screen
- Navigate with breadcrumbs
- Unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/routes/router.ts [hunk 1]
```diff
@@ -849,6 +851,21 @@ export function getRoutes (): RouteConfig[] {
         allowedRoles: [Role.CreateCredits, Role.FasRefund]
       }
     },
+    {
+      path: '/staff/involuntary-dissolution',
+      name: 'involuntary-dissolution',
+      component: InvoluntaryDissolution,
+      props: true,
+      meta: {
+        allowedRoles: [Role.Staff],
```
Comment (JazzarKarim) [inline auth-web/src/routes/router.ts:860]: This is a staff only screen. So, it's going to have only staff as the allowed roles.

File: auth-web/src/views/auth/staff/InvoluntaryDissolution.vue [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<template>
```
Comment (JazzarKarim) [inline auth-web/src/views/auth/staff/InvoluntaryDissolution.vue:1]: This is a preliminary skeleton for the Involuntary Dissolution screen. The actual screen will be worked on in a separate ticket after the designs have been finalized.

File: auth-web/src/views/auth/staff/StaffDashboardView.vue [hunk 3]
```diff
@@ -226,17 +226,54 @@
         <SafeEmailView />
       </template>
     </BaseVExpansionPanel>
+
+    <!-- Involuntary Dissolution -->
+    <v-card
+      v-if="showInvoluntaryDissolutionTile"
```
Comment (JazzarKarim) [inline auth-web/src/views/auth/staff/StaffDashboardView.vue:232]: Only show the tile when FF is true. This is tile will be shown for staff only.

File: auth-web/src/views/auth/staff/StaffDashboardView.vue [hunk 4]
```diff
@@ -287,24 +324,27 @@ export default defineComponent({
...
+      canViewGLCodes: computed((): boolean => currentUser.value?.roles?.includes(Role.ManageGlCodes)),
+      canViewIncorporationSearchResult: false,
+      errorMessage: '',
       isFasDashboardEnabled: computed((): boolean => currentUser.value?.roles?.includes(Role.FasSearch)),
+      registrySearchUrl: computed((): string => ConfigHelper.getRegistrySearchUrl()),
+      searchActive: false,
+      searchedBusinessNumber: '',
       showBusSearchlink: computed((): boolean => true),
-      registrySearchUrl: computed((): string => ConfigHelper.getRegistrySearchUrl())
+      showInvoluntaryDissolutionTile: computed((): boolean => LaunchDarklyService.getFlag(LDFlags.EnableInvoluntaryDissolution) || false)
```
Comment (JazzarKarim) [inline auth-web/src/views/auth/staff/StaffDashboardView.vue:341]: I just alphabetized these and added this here. This is the computed value of the involuntary dissolution FF.

File: auth-web/src/views/auth/staff/StaffDashboardView.vue [hunk 5]
```diff
@@ -354,13 +394,14 @@ export default defineComponent({
     }
 
     return {
-      searchBusinessForm,
       businessIdentifierRules,
-      isFormValid,
-      goToManageBusiness,
-      search,
       formatBusinessIdentifier,
+      goToInvoluntaryDissolution,
```
Comment (JazzarKarim) [inline auth-web/src/views/auth/staff/StaffDashboardView.vue:399]: Again, I just alphabetized these and added this here.

File: auth-web/tests/unit/views/InvoluntaryDissolution.spec.ts [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import { Wrapper, createLocalVue, mount } from '@vue/test-utils'
```
Comment (JazzarKarim) [inline auth-web/tests/unit/views/InvoluntaryDissolution.spec.ts:1]: Preliminary test of the view. We're going to probably add more stuff in the future when the actual view has been implemented.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2769-h9rlz5ic.web.app

**Review by ochiu — APPROVED** [review]

File: auth-web/src/views/auth/staff/StaffDashboardView.vue [hunk 7]
```diff
@@ -226,17 +226,54 @@
...
+        align="center"
+        justify="space-between"
+      >
+        <v-col>
+          <header>
+            <h2 class="mb-0">
+              Involuntary Dissolution
+            </h2>
+            <p class="mt-3 mb-0">
+              Set up and manage automation for Involuntary Dissolution of businesses
```
Comment (Jxio) [inline auth-web/src/views/auth/staff/StaffDashboardView.vue:246]: Could you double check with UX the wording here?
here is what I found on current design:
image

File: auth-web/src/views/auth/staff/StaffDashboardView.vue [hunk 8]
```diff
@@ -226,17 +226,54 @@
...
+          <v-btn
+            id="involuntary-dissolution-button"
+            class="mr-4"
+            color="primary"
+            outlined
+            dark
+            large
+            @click="goToInvoluntaryDissolution()"
+          >
+            <span>Manage Involuntary Dissolution</span>
```
Comment (Jxio) [inline auth-web/src/views/auth/staff/StaffDashboardView.vue:260]: same as above, wording

File: auth-web/src/views/auth/staff/StaffDashboardView.vue (hunk 7 above)
Comment (JazzarKarim) [reply auth-web/src/views/auth/staff/StaffDashboardView.vue:246]: Good catch Jia! Yes, you're right but the wording has changed. Andy has made changes to the design:
andy changes

So, I had the text the same as his latest changes.

**Review by Jxio — APPROVED** [review]

**Review by seeker25 — APPROVED** [review]

**Pull request merged**